### PR TITLE
Add Link functionality to FormButton component

### PR DESCRIPTION
**Pull request by marcinbodnar**

### CODE REVIEW

**Comment by vgrichina** [comment]
@marcinbodnar make sure to resolve conflicts

**Comment by marcinbodnar** [comment]
@vgrichina Done. I always rebase before PR. Is there a way to receive an email notification if conflicts appear after PR was opened?

File: src/components/dashboard/DashboardKeys.js [hunk 1]
```diff
@@ -70,11 +69,13 @@ const DashboardKeys = ({ image, title, to, accessKeys, empty }) => (
         <Grid.Row>
             <Grid.Column textAlign='left' width={16}>
                 {accessKeys && accessKeys.length !== 0 ? (
-                    <Link to={to}>
-                        <FormButton color='gray-blue' size='small'>
-                            <Translate id='button.viewAll' />
-                        </FormButton>
-                    </Link>
+                    <FormButton
+                        to={to}
```
Comment (Patrick1904) [inline src/components/dashboard/DashboardKeys.js:73]: Would it make sense to name URL prop something else? `to` doesn't help me understand what it is outside of FormButton context.

**Comment by Patrick1904** [comment]
This is a big out of context but does your IDE complain about lack of braces too?
Screen Shot 2020-04-08 at 1 10 51 PM

File: src/components/common/FormButton.js [hunk 2]
```diff
@@ -321,13 +322,19 @@ const FormButton = ({
...
-    size = ''
+    size = '',
+    to,
+    history,
+    className
 }) => (
     <CustomButton
         type={type}
-        className={`${color} ${size} ${sending ? `dots` : ``}`}
+        className={`${color} ${size} ${sending ? `dots` : ``} ${className}`}
```
Comment (Patrick1904) [inline src/components/common/FormButton.js:332]: Screen Shot 2020-04-08 at 1 23 42 PM

I think we should avoid `undefined` if no className?

**Comment by vgrichina** [comment]
> @vgrichina Done. I always rebase before PR. Is there a way to receive an email notification if conflicts appear after PR was opened?

not sure if you can receive e-mail notification, but I think you can get notification in Slack (https://github.com/pullreminders/backlog/issues/42) from http://pullreminders.com

File: src/components/common/FormButton.js (hunk 2 above)
Comment (vgrichina) [reply src/components/common/FormButton.js:332]: also maybe we should also remove  dependency on `import { Button } from 'semantic-ui-react'` and avoid dealing with `className` here altogether? 

otherwise looks like good use for `utils/classNames`

File: src/components/dashboard/DashboardKeys.js (hunk 1 above)
Comment (vgrichina) [reply src/components/dashboard/DashboardKeys.js:73]: `linkTo`?

File: src/components/common/FormButton.js (hunk 2 above)
Comment (Patrick1904) [reply src/components/common/FormButton.js:332]: Agree. Let's just make sure we test a lot if we remove semantic-ui since there are a lot of variations of the `FormButton` that may be affected.

Good point on `classNames`

**Comment by marcinbodnar** [comment]
@Patrick1904 Done.

> This is a bit out of context but does your IDE complain about lack of braces too?

It does, but it was fixed at the same time so I left it:
https://github.com/nearprotocol/near-wallet/pull/466/commits/74060e214e98d81c79132240ed75870d4499e1d5

Also added small change for `ClassNames`, so it will handle undefined `names`.

File: src/components/common/FormButton.js (hunk 2 above)
Comment (marcinbodnar) [reply src/components/common/FormButton.js]: >avoid dealing with className here altogether?

@vgrichina what do you mean?

Comment (vgrichina) [reply src/components/common/FormButton.js]: I mean that classes need to be set only because we rely on semantic-ui-react.  If not using semantic-ui-react (creating our custom component from scratch) we can for example use props directly in styled-components CSS (or otherwise apply styles programmatically).

File: src/utils/classNames.js [hunk 3]
```diff
@@ -3,6 +3,10 @@
     <div className={classNames(['base-class', {'second' : someFlag, 'third' : true}])}></div>
 */
 const classNames = (names) => {
+    if (!names) {
+        return false
```
Comment (vgrichina) [inline src/utils/classNames.js:7]: nit: seems like should be returning '' instead?

File: src/utils/classNames.js [hunk 4]
```diff
@@ -3,6 +3,10 @@
     <div className={classNames(['base-class', {'second' : someFlag, 'third' : true}])}></div>
 */
 const classNames = (names) => {
+    if (!names) {
```
Comment (vgrichina) [inline src/utils/classNames.js:6]: nit: @marcinbodnar makes sense to add test if you find some bug with edge case (given this already has tests)
